bin/report

```diff
@@ -141,10 +145,14 @@ kv_pair "npm_install_time" "$(meta_get "npm-install-time")"
 kv_pair "npm_install_memory" "$(meta_get "npm-install-memory")"
 kv_pair "yarn_install_time" "$(meta_get "yarn-install-time")"
 kv_pair "yarn_install_memory" "$(meta_get "yarn-install-memory")"
-kv_pair "npm_prune_time" "$(meta_get "yarn-prune-time")"
```

Comment:
`"npm_prune_time" != "yarn-prune-time"` 😱 -  Looks like this has been reported incorrectly for a while. Thanks for fixing it.